packages/config/src/Config.ts

```diff
@@ -121,6 +121,8 @@ export interface Config {
   registries: Registries
   ignoreWorkspaceRootCheck: boolean
   workspaceRoot: boolean
+
+  testPattern?: string
```

Comment:
It should probably be an array of strings

Reply:
How would you expect a declaration of an array in `.npmrc` to look like?
Can you provide an example of a property that is defined as an array of strings?

Reply:
```
test-pattern[]=*.test.ts
```

Reply:
I'm still not sure I completely understand. how would multiple values look like? Is it something the npm config loader we're using already supports?

Reply:
done [here](https://github.com/pnpm/pnpm/pull/3035/commits/cb55a1189913192ce8755ff5ae6c56976866219b)

used `public-hoist-pattern` as an example.

do you prefer to rename the property to testPattern**s** ?